tests/integration/test_ols.py

```diff
@@ -167,31 +166,6 @@ def test_post_question_on_generic_response_type_summarize_error():
             assert response.json() == expected_json
 
 
-def test_post_question_on_generic_response_llm_configuration_error():
```

Comment:
it is concerning that we managed to write a test that successfully verified that our code was doing something that our code doesn't actually do.  This is the kind of thing that worries me w/ our heavily mocked tests.

thanks for cleaning it up

Reply:
yeah, it should be part of DocSummarizer refactoring - but PR with this refactoring did not touch tests for some reason